processing/pom.xml

```diff
@@ -44,6 +44,10 @@
             <artifactId>emitter</artifactId>
         </dependency>
         <dependency>
+            <groupId>com.metamx</groupId>
```

Comment:
@xvrl yeah that's my bad

@binlijin you can remove the exclusion from the code